shelley/chain-and-ledger/formal-spec/chain.tex

```diff
@@ -722,10 +726,12 @@ \subsection{Block Header Transition}
       \var{s_\ell} < \var{slot} \leq \var{s_{now}}
       &
       \var{h} = \bprev{bhb}
-      \\
-      \mathcal{V}_{vk_{hot}}{\serialised{bhb}}_{\sigma}^{t}
       &
+      \mathcal{V}_{vk_{hot}}{\serialised{bhb}}_{\sigma}^{t}
+      \\
       \bHeaderSize{bh} \leq \fun{maxBHSize}~\var{pp}
+      &
+      \bhbodysize{bhb} < \fun{maxBBSize}~\var{pp}
```

Comment:
why is this strictly less ?

Reply:
good catch, I think `<=` makes more sense